tests/shared.py

```diff
@@ -29,6 +29,22 @@ def new_backend():
     return Backend(host, port)
 
 
+def driver_feature(capability):
+    def get_valid_test_case(*args, **kwargs):
+        if not isinstance(args[0], TestkitTestCase):
```

Comment:
```python
if not args or not isinstance(args[0], TestkitTestCase):
```
To handle the case if someone decorates a function that is called with no arguments.

Reply:
Thanks, addressed.